cucumber/features/steps/activityReportStepperSteps.js

```diff
@@ -0,0 +1,158 @@
+
+  const value = await page.$eval(selector, (el) => el.textContent);
+
+  assertTrue(value);
+  assertTrue(value.startsWith('Activity Summary'));
+});
+
+Then('the first step is the Activity Summary', async () => {
+  const page = scope.context.currentPage;
+  const selector = '[aria-label="progress"] ol li span[aria-current="true"]';
+
+  const value = await page.$eval(selector, (el) => el.textContent);
+
+  assert.equal(value, 'Activity Summary');
+});
+
+Then('I see two navigation buttons', async () => {
+  const page = scope.context.currentPage;
+
+  const buttonOne = await select(page).getElement('button:contains("Previous")');
+  const buttonTwo = await select(page).getElement('button:contains("Next")');
+
+  assertTrue(buttonOne);
+  assertTrue(buttonTwo);
+
+  let value = await page.evaluate((el) => el.textContent, buttonOne);
+  assert.equal(value, 'Previous');
+
+  value = await page.evaluate((el) => el.textContent, buttonTwo);
+  assert.equal(value, 'Next');
+});
+
+Then('the {string} button is disabled', async (string) => {
+  const page = scope.context.currentPage;
+  const buttonOneSelector = 'button[disabled]';
+  // const buttonOne = await select(page).getElement('button:contains("Previous")');
+  // const buttonTwo = await select(page).getElement('button:contains("Next")');
+  const buttonPrevious = await page.$(buttonOneSelector);
+
+  assertTrue(buttonPrevious);
+  // assertTrue(buttonTwo);
+
+  const value = await page.evaluate((el) => el.textContent, buttonPrevious);
+  assert.equal(value, string);
+
+  // value = await page.evaluate((el) => el.disabled, buttonTwo);
+  // assert.equal(value, 'Next');
+});
+
+When('I click the {string} button', async (string) => {
+  const page = scope.context.currentPage;
+  const buttonTwo = await select(page).getElement(`button:contains(${string})`);
+  await buttonTwo.click();
+});
+
+Then('the {string} button is no longer disabled', async (string) => {
+  const page = scope.context.currentPage;
+  const buttonOneSelector = 'button[disabled]';
+
+  const buttonPrevious = await page.$(buttonOneSelector);
+
+  const value = await page.evaluate((el) => el, buttonPrevious);
+  assert.equal(value, null);
+});
+
+Then('I moved past the {string} step', async (string) => {
+  const page = scope.context.currentPage;
+  assert.equal(string, 'Activity Summary');
+  const selector = `[data-testid="${string}"] > [aria-current="false"]`;
+
+  const value = await page.$eval(selector, (el) => el.textContent);
+
+  assertTrue(value);
+  assertTrue(value.startsWith(string));
+});
+
+Then('I am on the {string} step', async (string) => {
+  const page = scope.context.currentPage;
+  // assert.equal(string, 'Participants');
+  const selector = `[data-testid="${string}"] > [aria-current="true"]`;
+
+  const value = await page.$eval(selector, (el) => el.textContent);
+
+  assertTrue(value);
+  assert.equal(value, string);
+});
+
+When('I click the {string} button again', async (string) => {
```

Comment:
Can we remove this `When` and just use the `I click the {string} button`?

Reply:
Same as above. I'll be happy to revisit this once we bring the stepper back.